.readthedocs.yml

```diff
@@ -1,9 +1,8 @@
-version: 2
-
-sphinx:
-  configuration: docs/source/conf.py
-python:
-  version: 3.7
-
-conda:
-  environment: docs/source/rtd_environment.yml
+---
+version: 2
+sphinx:
+  configuration: docs/source/conf.py
+python:
+  version: 3.7
```

Comment:
While we are at it, this should be updated.

Reply:
I have not even tried building the docs, would push that off for the moment. Still updated it.